rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -539,6 +539,52 @@ void setLockWhitelistDisableBlockDelay() throws BlockStoreException, IOException
         }
     }
 
+    @Nested
+    @Tag("LockingCap")
+    class LockingCapTest {
+
+        private LockingCapSupport lockingCapSupport;
+        private BridgeSupport bridgeSupport;
+        LockingCapConstants constants = LockingCapMainNetConstants.getInstance();
+
+        @BeforeEach
+        void setUp() {
+            lockingCapSupport = mock(LockingCapSupportImpl.class);
+            bridgeSupport = bridgeSupportBuilder
+                .withLockingCapSupport(lockingCapSupport)
+                .build();
+        }
+
+        @Test
+        void getLockingCap() {
```

Comment:
one test for when the value from lockingCapSupport is empty? assert that bridgeSupport.getLockingCap() is null. wdyt?

Reply:
Same

Reply:
I don't think this new test can add some value . @julia-zack, the previous refactors, the tests added to the `BridgeSupportTest` were kind of simple and straight to the point to add coverage without thinking of making a list of test cases for each method since they were covered in `Support` test classes. WDYT?

Reply:
well, considering that the implementation in [BridgeSupport](https://github.com/rsksmart/rskj/blob/feature/bridgesupport-lockingcap-refactor-test/rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java#L2116) actually returns something different when the value got from locking cap support is null
(code: `return lockingCapSupport.getLockingCap().orElse(null);`)
i think the test adds value.

Reply:
Done!